app/controllers/orga/teams_controller.rb

```diff
@@ -10,6 +10,9 @@ def index
     else
       @teams = Team.order(:kind, :name)
     end
+    if params[:show] != 'all_teams'
```

Comment:
Almost purely cosmetic, but can we change this to :filter or something? :show clashes with the controller action. Not logically, just in my brain 🙈 😉